archlinux/dist

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -o errexit -o pipefail
+
+export cpuCount=$(awk '/^processor/{n+=1}END{print n}' /proc/cpuinfo)
+git submodule update --init --recursive
+mkdir -p build
+cd build && cmake .. \
```

Comment:
I think this can be simply split into two lines, one for `cd build` and the other for `cmake ...`, as these commands are all always expected to succeed.

Reply:
Will change that to make it cleaner.

Reply:
Changed it.